AnnaLoginova/homework1.js

```diff
@@ -0,0 +1,34 @@
+// Create function to check is the string/number a palindrome
+function reverseString(str) {
+    return str.split('').reverse().join('');
+}
+function isPalindrome(value) {
+    const str = value.toString();
+    const firstHalfStr = str.substring(0, Math.floor(str.length/2));
+    const secondHalfStr = str.substring(Math.ceil(str.length/2), str.length);
+    return (firstHalfStr.toLowerCase() === reverseString(secondHalfStr.toLowerCase()));
+}
+console.log('------ Check Palindrome function ------');
+
+console.log('"ghJ4jhg" is ' + isPalindrome('ghJ4jhg'));
```

Comment:
I would add unit tests for instead of these `console.log`, but it's not necessary :)